app/assets/javascripts/angular/controllers/mobile/base-mobile.js

```diff
@@ -0,0 +1,48 @@
+(function() {
```

Comment:
This is the base Mobile controller. While many of our controllers run repeatedly (check income, check messages, check terror), this one only runs once and before all of the others. It gets things that we know won't change - the user's team, their role on the team, and their permissions in that role.